interface/src/app/formatMissing.filter.js

```diff
@@ -0,0 +1,14 @@
+angular.module('adage.formatMissing.filter', [])
+
+.filter('formatMissing', function() {
+  return function(inputStr, replacementStr) {
+    // default to 'N/A'
+    var replacement = (replacementStr === undefined ? 'N/A' : replacementStr);
```

Comment:
I got a little confused about the two variables "replacement` and `replacementStr`. Can you rename one of them?

Reply:
Good point. Will do that.